packages/story-editor/src/components/panels/shared/backgroundAudioPanelContent.js

```diff
@@ -197,6 +198,8 @@ function BackgroundAudioPanelContent({
               )}
               src={resource?.src}
               mimeType={resource?.mimeType}
+              tracks={tracks || []}
```

Comment:
`tracks` already defaults to an empty array further above on line 72:

```js
const { resource, tracks = [], loop = true } = backgroundAudio || {};
```

So this shouldn't be needed here